### PR TITLE
design layout html

### PR DESCRIPTION
**Pull request by st-ngabui**
**Description**
- Design layout html from website template

**Dependence**
- #4

### CODE REVIEW

File: exersice_html/home.html [hunk 1]
```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Shopka</title>
+</head>
+<body>
+
```
Comment (st-hahoang) [inline exersice_html/home.html:10]: Remove empty line.

File: exersice_html/home.html [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        <defs>
+        <linearGradient id="paint0_linear" x1="12" y1="40" x2="52" y2="40" gradientUnits="userSpaceOnUse">
+        <stop stop-color="#2979FF"/>
+        <stop offset="1" stop-color="#4C589E"/>
+        </linearGradient>
+        </defs>
+        </svg>
+        
+    </div>
+    <div>
```
Comment (st-hahoang) [inline exersice_html/home.html:26]: Bỏ những div dư.

File: exersice_html/home.html [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  <footer class="my-footer">
+    <div class="footer-wrap">
+      <p class="footer-text-top">Are you ready</p>
+      <p class="footer-text-middle">Let's get started</p>
+      <p class="footer-text-bottom">Copyright © 2020. LogoIpsum. All rights reserved.</p>
+    </div>
+  </footer>
+</body>
+
+</html>
```
Comment (st-quocpham) [inline exersice_html/home.html:239]: always need an empty line at the end. It's a rule

File: exersice_html/home.html [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Shopka</title>
+</head>
+
+<body>
+  <header class="my-header">
+    <div class="header-wrap">
+      <div class="header-left">
```
Comment (st-quocpham) [inline exersice_html/home.html:14]: should be same `header` as initial word (not `my-header`), will be useful when using scss/sass

**Pull request merged**